src/components/Settings.vue

```diff
@@ -125,6 +125,7 @@
                         hint="Path to devices library DB. If not set the default path will be used based on your OS"
                       ></v-text-field>
                     </v-flex>
+                    <input type="hidden" :value="zwave.plugin">
```

Comment:
This is not necessary you can remove it :)

Reply:
without it, hitting save in the settings removed it from the `settings.json`

Reply:
Oh ok, no problem so. Keep it :)